message_definitions/v1.0/common.xml

```diff
@@ -1471,6 +1474,16 @@
         <param index="6">Empty</param>
         <param index="7">Empty</param>
       </entry>
+      <entry value="198" name="MAV_CMD_DO_SET_ROI_SYSID">
+        <description>Follow vehicle by system ID.</description>
+        <param index="1" label="System ID" minValue="0" maxValue="255" increment="1">sysid</param>
+        <param index="2">Empty</param>
```

Comment:
Just FYI, you don't have to declare the empty parameters. In this case the toolchain will just create a bunch of them with name Reserved.

Reply:
Removed.